docs/programming-oak.md

```diff
@@ -574,20 +656,60 @@ framework via the Oak Runtime:
 async fn test_say_hello() {
     let _ = env_logger::builder().is_test(true).try_init();
 
-    let runtime = oak_tests::run_single_module_default(MODULE_WASM_FILE_NAME)
+    let runtime_config = oak_tests::runtime_config_wasm(
+        hashmap! {
+            MAIN_MODULE_NAME.to_owned() => oak_tests::compile_rust_wasm(MAIN_MODULE_MANIFEST, MAIN_MODULE_WASM_FILE_NAME, oak_tests::Profile::Debug).expect("Couldn't compile main module"),
+            TRANSLATOR_MODULE_NAME.to_owned() => oak_tests::compile_rust_wasm(TRANSLATOR_MODULE_MANIFEST, TRANSLATOR_MODULE_WASM_FILE_NAME, oak_tests::Profile::Debug).expect("Couldn't compile translator module"),
+        },
+        MAIN_MODULE_NAME,
+        MAIN_ENTRYPOINT_NAME,
+        ConfigMap::default(),
+    );
+    let runtime = oak_runtime::configure_and_run(runtime_config)
```

Comment:
Personally, I'd still lean towards having an initial section that illustrates single-Wasm-module testing using `oak_tests::run_single_module` (e.g. using `examples/translator/module/tests/integration_test.rs` as the code fragment), then deferring something that's multi-Wasm-module (e.g. `hello_world` with this PR) to a second subsection.

Reply:
Done.